app/src/main/java/com/hedvig/app/util/compose/ModifierExt.kt

```diff
@@ -0,0 +1,22 @@
+package com.hedvig.app.util.compose
+
+import android.view.KeyEvent
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.input.key.onKeyEvent
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+
+@OptIn(ExperimentalComposeUiApi::class)
+fun Modifier.submitOnEnter(action: () -> Unit) = composed {
```

Comment:
Closing the keyboard is more of a "hidden side effect" behind a pretty innocent looking name on the function. Might catch us unprepared sometime in the future when we forget it also does that. Do you think it's important to denote that it does that in the name?

Reply:
I'd consider it par for the course when submitting using `Enter`, but perhaps it's not? 🤔

Reply:
Hmm yeah fair point, Enter kinda implies that, as it's the default keyboard behaviour.